manim/utils/space_ops.py

```diff
@@ -231,10 +343,21 @@ def angle_of_vector(vector):
         return np.angle(complex(*vector[:2]))
 
 
-def angle_between_vectors(v1, v2):
-    """
-    Returns the angle between two 3D vectors.
+def angle_between_vectors(v1: Sequence[float], v2: Sequence[float]) -> float:
```

Comment:
Is this really a sequence of floats? Shouldn't it be a np.ndarray?

Reply:
I was thinking it can be both of them